templates/components/settings/python/settings.py

```diff
@@ -127,8 +127,9 @@ def init_groq():
     from llama_index.llms.groq import Groq
 
     model_map: Dict[str, str] = {
-        "llama3-8b": "llama3-8b-8192",
-        "llama3-70b": "llama3-70b-8192",
+        "llama3.1-8b": "llama-3.1-8b-instant",
```

Comment:
why deleting llama3?

Reply:
Ah I mean no real reason just because there are the 3.1 of the same size but newer. But I mean for back compatibility we can definitely keep the old lama 3 models

Reply:
I can add it back if you want.

Reply:
@Jac-Zac sorry for the late reply, how about removing this mapping at all and instead retrieve the full model name in https://github.com/run-llama/create-llama/blob/main/helpers/providers/groq.ts
(by using a function similar to https://github.com/run-llama/create-llama/blob/8ce4a8513d9889427f80a7b5d1ce7bc04f35932f/helpers/providers/openai.ts#L89C1-L138C2)
endpoint to list the groq models is https://api.groq.com/openai/v1/models (see https://console.groq.com/docs/models)

Reply:
I was looking at doing something like this but lately, I do not have much time sorry.